addons/web/static/src/core/file_input/file_input.js

```diff
@@ -52,8 +52,8 @@ export class FileInput extends Component {
     async uploadFiles(params) {
         const fileData = await this.http.post(this.props.route, params, "text");
         const parsedFileData = JSON.parse(fileData);
-        if (parsedFileData.error) {
-            throw new Error(parsedFileData.error);
+        if (parsedFileData.length && parsedFileData[0].error) {
```

Comment:
`uploadFiles` could upload many files. If the second file caused an error, it should also throw an error (not only the first one)